config/config.go

```diff
@@ -55,8 +60,8 @@ type (
 		// InternalHttpServer is the config for starting a http.Server
```

Comment:
maybe rename and/or add some comments to explain this vs `AsyncServiceAddresses`. This is for binding, where `AsyncServiceAddresses` is for resolving. 

Maybe this is `InternalAddressToBind`

Reply:
I think current desp is clear to tell this is with HTTP config.